Source/DafnyCore/AST/Expressions/Operators/BinaryExpr.cs

```diff
@@ -323,6 +323,8 @@ public enum AccumulationOperand { None, Left, Right }
     if (cloner.CloneResolvedFields) {
       ResolvedOp = original.ResolvedOp;
     }
+
+    RangeToken = original.rangeToken;
```

Comment:
The call to base already does `RangeToken = cloner.Range(original.RangeToken);`, so this line seems obsolete